templates/api/knowledge-panels/health/ingredients/nova.tt.json

```diff
@@ -0,0 +1,59 @@
+{
+    "level": "info",
+    "topics": [
+        "health"
+    ],
+    "title_element": {
+        [% IF product.nova_group > 0 %]
+            [% SET nova_group = product.nova_group %]
+            "icon_url": "[% static_subdomain %]/images/attributes/nova-group-[% product.nova_group %].svg",
+            "title": "[% lang('attribute_nova_' _ nova_group _ '_description_short') %]",
+        [% ELSE %]
+            "icon_url": "[% static_subdomain %]/images/attributes/nova-group-unknown.svg",
+            "title": "[% lang('attribute_nova_unknown_description_short') %]",       
+        [% END %]
+    },
+    "elements": [
+        {
+            "element_type": "text",
+            "text_element": {
+                "html": `
+[% IF ! product.nova_group.defined %]
+    [% IF product.nova_group_error.defined %]
+       [% lang("nova_group_" _ product.nova_group_error) %]
+    [% END %]
+[% ELSE %]
+    [% IF nova_group == 1 %]
+    [% ELSIF nova_group >= 2 %]
+        [% IF product.nova_groups_markers.$nova_group.defined %]
```

Comment:
this part seems broken, I have nothing displayed (but I enter the IF, I put a simple string to be sure)

Reply:
I forgot to mention that the NOVA analysis has to be done again on the product for the nova markers to be computed. If you edit and save the product, that should be enough.